jupyterlab_git/git.py

```diff
@@ -497,6 +497,15 @@ async def branch(self, current_path):
             "current_branch": heads["current_branch"],
         }
 
+    async def branch_delete(self, current_path, branch):
+        """Execute 'git branch -D <branchname>'"""
+        cmd = ["git", "branch", "-D", branch]
+        code, _, error = await execute(
+            cmd, cwd=os.path.join(self.root_dir, current_path)
+        )
+        if code != 0:
+            return {"code": code, "command": " ".join(cmd), "message": error}
+
```

Comment:
I think you also need to return sometime in the case of success. Otherwise the `if result['code']` in the handler will fail
```suggestion
        if code != 0:
            return {"code": code, "command": " ".join(cmd), "message": error}
        else:
            return {"code": 0}

```